xml/mysql_banners.xml

```diff
@@ -0,0 +1,850 @@
+    <param pos="0" name="os.product" value="Linux"/>
+  </fingerprint>
+
+  <fingerprint pattern="^(\d\.\d{1,2}\.\d{1,2})(?:-rc)?[_-]Debian[_-]\d{1,2}~ppa\d(?:-log)?$">
+    <example>5.1.22-rc-Debian_2~ppa5-log</example>
+    <description>Oracle MySQL on Debian (Generic match) using PPA</description>
+    <param pos="1" name="service.version"/>
+    <param pos="0" name="service.vendor" value="Oracle"/>
+    <param pos="0" name="service.family" value="MySQL"/>
+    <param pos="0" name="service.product" value="MySQL"/>
+    <param pos="0" name="os.vendor" value="Debian"/>
+    <param pos="0" name="os.device" value="General"/>
+    <param pos="0" name="os.family" value="Linux"/>
+    <param pos="0" name="os.product" value="Linux"/>
+  </fingerprint>
+
+  <fingerprint pattern="^(\d\.\d{1,2}\.\h{1,3}(?:-\d{1,2})?)(?:-Debian)?(?:-Dotdeb)?[-~_\d\.]+dotdeb\.\d(?:-log)?$">
+    <example>5.1.54-0.dotdeb.0</example>
+    <example>5.1.58-1~dotdeb.0</example>
+    <example>5.5.30-1~dotdeb.0-log</example>
+    <example>5.1.30-Dotdeb_0.dotdeb.0-log</example>
+    <description>Oracle MySQL packaged by DotDeb.org</description>
+    <param pos="1" name="service.version"/>
+    <param pos="0" name="service.vendor" value="Oracle"/>
+    <param pos="0" name="service.family" value="MySQL"/>
+    <param pos="0" name="service.product" value="MySQL"/>
+    <param pos="0" name="os.vendor" value="Debian"/>
+    <param pos="0" name="os.device" value="General"/>
+    <param pos="0" name="os.family" value="Linux"/>
+    <param pos="0" name="os.product" value="Linux"/>
+  </fingerprint>
+
+  <fingerprint pattern="^(\d\.\d{1,2}\.\h{1,3})(?:-alpha)?(?:-beta)?(?:-gamma)?(?:-rc)?-standard(?:-log)?$">
+    <example>5.0.13-rc-standard-log</example>
+    <example>5.0.37-standard</example>
+    <example>5.0.51a-standard-log</example>
+    <description>Oracle MySQL Standard Edition</description>
+    <param pos="1" name="service.version"/>
+    <param pos="0" name="service.vendor" value="Oracle"/>
+    <param pos="0" name="service.family" value="MySQL"/>
+    <param pos="0" name="service.product" value="MySQL"/>
+    <param pos="0" name="service.edition" value="Standard Edition"/>
+  </fingerprint>
+
+  <fingerprint pattern="^(\d\.\d{1,2}\.\h{1,3})-pro(?:-gpl)?-nt(?:-log)?$">
+    <example>5.0.24-pro-nt</example>
+    <example>4.1.25-pro-gpl-nt-log</example>
+    <description>Oracle MySQL Commercial Edition with Named Pipes (Windows)</description>
+    <param pos="1" name="service.version"/>
+    <param pos="0" name="service.vendor" value="Oracle"/>
+    <param pos="0" name="service.family" value="MySQL"/>
+    <param pos="0" name="service.product" value="MySQL"/>
+    <param pos="0" name="service.edition" value="Commercial Edition"/>
+    <param pos="0" name="os.vendor" value="Microsoft"/>
+    <param pos="0" name="os.device" value="General"/>
+    <param pos="0" name="os.family" value="Windows"/>
+    <param pos="0" name="os.product" value="Windows"/>
```

Comment:
os.family alone is sufficient in this case.  If you could tell more specifically what version of Windows, great, by Windows alone as a product is too vague.